app/src/main/java/com/kickstarter/ui/viewholders/EditorialViewHolder.kt

```diff
@@ -29,11 +26,6 @@ class EditorialViewHolder(val view: View, val delegate: Delegate) : KSViewHolder
                 .compose(Transformers.observeForUI())
                 .subscribe { this.itemView.description.setText(it) }
 
-        this.vm.graphic()
```

Comment:
No needed as it has to be an ImageView, this code was from the version where it was a constraintlayout